.gitignore

```diff
@@ -1,14 +1,11 @@
 # Hugo-generated assets
-public/
+/public
```

Comment:
This change allows us to ignore the top-level `public` directory (which contains the generated site files), while allowing `/public` to be a symlink (e.g., to another git tracked repo -- a trick that is very useful for examining diffs of generated site files).